imod_coupler/drivers/dfm_metamod/msw_wrapper.py

```diff
@@ -0,0 +1,122 @@
+from typing import Optional
+
+import numpy as np
+from numpy.typing import NDArray
+from xmipy import XmiWrapper
+
+
+class MswWrapper(XmiWrapper):
+    def initiate_surface_water_component(self) -> None:
+
+        """
+        function to initialise the surface water component in metaswap.
+
+        Parameters
+        ----------
+        none
+
+        """
+        self.initiate_surface_water_component()
```

Comment:
sonarqube has a comment about this function: it calls itself and will get stuck in an infinite loop.